docs/user/faq/troubleshooting.md

```diff
@@ -2,106 +2,118 @@
-
-## Why can't I connect via VPN to my DAppNode?
-
-If you have set up openvpn and you cant access your dappnode, remember you need to set up the port forwarding in your router. To do this, you need to access your router configuration and "open" the next ports: TCP port 8092 and 1194 UDP. Sometimes the UPnP of the router enable these ports automatically but another does not and it's required to do this.
-
-In case you have set up wireguard vpn, you have to create two profiles:
-
-- Local: This profile is used when you are connected from the same network that the dappnode.
-- Remote: This profile is used when you are connected from a different network that the dappnode.
-  When you create the vpn profile there is a link that generate the profile for local or remote.
-
-## Can I recover my admin password?
-
-You can't recover your admin password, but you can access to your dappnode if you wrote down the recovery token, and then change the admin password.
-
-The first time you access your DAppnode,it appears the next image:
-
-<p align="center">
-    <img src="../../../img/sign_in_image.png"/>
-</p>
-
-Once you fill the fields and press the register button, DAppNode provides you with a recovery code. If you forget te DAppNode's Password you will be able to access using this token.
-
-<p align="center">
-    <img src="../../../img/recovery_token.png"/>
-</p>
-
-You should save your token in a secure place like a password manager, paper, etc. In this case, we would save this:
-
-```
-4LMB9w3l50Yljwr6bIgQ
-```
-
-## Can I recover my recovery token?
-
-Firstly, we have to go to the login page of our DAppNode:
-
-<p align="center">
+<details>
+  <summary>Help! I can't access my DAppNode!</summary>
+  <div>
+    <div>
+    If it is your first time trying to access your DAppNode, please try one of the following steps:
+    <br /><br />
+<ul>
+    <li>If you are connected to the same network as your DappNode via wired Ethernet, try to access the UI using the address <a href="http://dappnode.local">http://dappnode.local</a></li>
+    <li>If you've bought a DAppNode or installed it on a machine with integrated Wi-Fi, try to access the Wi-Fi hotspot 'DAppNodeWifi'.</li>
+</ul>
+
+If all of the above does not work, visit our <a href="https://discord.com/invite/dappnode">Discord</a> and ask for help in the #support channel.
+</div>
+  </div>
+</details>
+
+<details>
+  <summary>Why can't I connect to my DAppNode via VPN?</summary>
+  <div>
+    <div>
+    If you have set up OpenVPN and you cant access your DAppNode, remember that you might need to set up port forwarding on your router. Sometimes the feature called 'UPnP' enables these ports on your router automatically but this sometimes does not work or is not available on your router.
+    <br /><br />
+
+To do this, you need to access your router configuration and "port-forward" the following ports: TCP Port 8092 and UDP Port 1194. 
+
+In case you have set up Wireguard, you might have to create two profiles:
```

Comment:
port forwarding issue occurs here too sometimes just less so so mention make sure port 51820 is open

Reply:
Agreed! Will add that, too!